docs/pages/desktop-access/reference/sessions.mdx

```diff
@@ -87,4 +76,8 @@ player supports toggling between play and pause, but does not support seeking to
 a specific point in the stream, rewinding, or restarting playback when the end
 of the stream is reached. To replay a session, refresh the page.
 
+## Log rotation
 
+Be aware, Desktop Access session recording comprises of raw PNG-based files.
```

Comment:
I think we should expand "comprises of" so it's clearer how these PNG-based files will affect your infrastructure/how they travel. Will the PNG-based files be stored uncompressed on the Teleport backend, or are they stored locally to each Desktop Service instance? Is this an issue for Teleport Cloud?

Reply:
I've updated this section based on internal discussion. Additional documentation on audit and session logs may be coming soon, which we can then link to here.